corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -584,16 +584,16 @@ def soft_delete_forms(domain, form_ids, deletion_date=None, deletion_id=None):
 
         return affected_count
 
-    @staticmethod
     @transaction.atomic
-    def _archive_unarchive_form(form, user_id, archive):
+    def _archive_unarchive_form(self, form, user_id, archive):
```

Comment:
this can be changed back to static

Reply:
https://github.com/dimagi/commcare-hq/pull/22470/commits/543f0c30a88142f1452ba2676affc6cca8aa2ddd